deploy-help/deploy.sh

```diff
@@ -0,0 +1,41 @@
+#! /bin/bash -e
+
+deployment_dir=~/deployment
+
+function ansible_command {
+    echo "playbook: '$1', cluster: '$2', cluster type: '$3'"
+    file="$1"
+    cluster_name="$2"
+    cluster_type="$3"
+    ansible-playbook                                                      \
+        $file                                                             \
+        --extra-vars "cluster=$cluster_name cluster_type=$cluster_type"   \
+        --inventory $deployment_dir/inventory
+}
+
+(
+    cd $deployment_dir/concent-deployment/concent-builder/
+    git fetch origin --prune
+    git checkout origin/dev
+
+    databases=(
+        control
+        storage
+    )
+    playbooks=(
+        install-repositories.yml
+        build-test-and-push.yml
+        job-cleanup.yml
```

Comment:
Ok. I change order of playbooks and put the `job-cleanup.yml` playbook as first playbook. 

We need two times execute this playbook. The first running is to make sure that kubernetes job with migrate-db isn't running on the cluster. The second running is to cleanup after first database is migrate, to allow migrate of second database.

Reply:
Oh, right, we have two separate migrate jobs with the same name. So let's add database name to the job name. Both these jobs are created from a template. It's just a matter of using the template variable there and then updating `clean-up-jobs.sh` to remove both jobs.

Reply:
I checked and `job-cleanup.yml` can't be executed as first playbook because it's depends on `build-test-and-push.yml`.

Reply:
True. We need to build it first. But it can still be before `deploy` rather than after it.

Reply:
I agree with you but it's already before `deploy`.

Reply:
Yeah, I'm not saying it's not :) I just wanted to clarify that this is what is really needed here - it does not have to be before everything (if fact it can't as you pointed out), but it has to be before `deploy` so that we can start deployment in a clean situation without worrying about job name conflicts.